docs/userguide/async.md

```diff
@@ -0,0 +1,74 @@
+could for instance be:
+- **Distributed workflows**: When recommendations are distributed across several
+  operators, e.g. at different locations or in several reactors, some experiments might
+  have been started, but are not ready when the next recommendation is requested.
+  Without further consideration, the algorithm would be likely to recommend the pending
+  measurements again, as it is unaware they were already started.
+- **Partial targets**: Relevant when dealing with multiple targets which require very
+  different amounts of time to measure. When a new recommendation is requested, the
+  targets of previously recommended points might only partially available, but that
+  info should ideally already be considered.
+
+
+## Pending Measurements
+
+With *pending measurements* we mean recommendations whose measurement process has
+been started, but not yet completed by time of triggering the next set of
+recommendations. This is not to be confused with a situation where the targets of a
+recommendation have been partially measured. A measurement is *pending* whenever one or
+more of its targets are not measured yet. Only once all the targets are available,
+the data point can be removed from `pending_measurements` and be provided via 
+`measurements`.
+
+To avoid repeated recommendations in the above scenario, BayBE provides the 
+`pending_measurements` keyword. It is available in `.recommend` from
+[`Campaign`](baybe.campaign.Campaign) or any
+[`Recommender`](baybe.recommenders.base.RecommenderProtocol).
+
+```{admonition} Supported Acquisition Functions
+:class: important
+`pending_measurementes` is only supported by Monte Carlo acquisition functions, i.e. the
+ones that start with a `q` in their name. Attempting to use a non-MC acquisition
+function with `pending_measurements` will result in a 
+[`IncompatibleAcquisitionFunctionError`](baybe.exceptions.IncompatibleAcquisitionFunctionError).
+```
+
+```{admonition} Supported Recommenders
+:class: important
+For technical reasons, not every recommender might be able to fully or even partially
+utilize `pending_measurements`. For instance, 
+[`BotorchRecommender`](baybe.recommenders.pure.bayesian.botorch.BotorchRecommender)
+takes all pending measurements into account, even if they do not match exactly with any
+point in the searchspace.
+[Clustering recommenders](baybe.recommenders.pure.nonpredictive.clustering.SKLearnClusteringRecommender)
+only incorporate pending points which are exact searchspace matches by excluding them
+from the candidates. The 
+[`RandomRecommender`](baybe.recommenders.pure.nonpredictive.sampling.RandomRecommender)
+does not take pending points into consideration at all and will raise a correspoding
+[warning](baybe.exceptions.UnusedObjectWarning).
+```
+
+Akin to `measurements` or `recommendations`, `pending_measurements` is a dataframe in
+experimental representation. In the following example, we get a set of recommendations,
```

Comment:
I think if you specifically mention "experimental representation", we should add a link to where this is explaine (i.e. search space?)

Reply:
still unable to dos o, please provide instructions

response pending 4w, 1 more day otherwise I will yank this comment

Reply:
There is also a way to use implicit target that are autogenerated for headers, but I also couldn't get it running quickly. I think we should start creating a sphinx/myst cheatsheet somewhere, because every time I want to reference something, I basically start my search from scratch. And I'm like 99% sure that I already had this running previously in some other context!

Anyway, as a workaround, let's just use an explicit anchor. You can write
`is a dataframe in [experimental representation](#DATA_REPRESENTATION)` and then just put `(DATA_REPRESENTATION)=` above the representation section. Have tested it and it works

Reply:
yes, tested this exact situation

Reply:
ok thanks for pointing it out, includede it (but currently cant verify lcoally due to docbuild hickups). willr esolve once confirmed